administration/src/mui-modules/download/DownloadPage.tsx

```diff
@@ -0,0 +1,65 @@
+import { Box, Typography } from '@mui/material'
+import MUILink from '@mui/material/Link'
+import React, { ReactElement, useContext } from 'react'
+import { useTranslation } from 'react-i18next'
+import { Link } from 'react-router'
+
+import AppStoreLinks from '../../bp-modules/components/AppStoreLinks'
+import { ProjectConfigContext } from '../../project-configs/ProjectConfigContext'
+import { getBuildConfig } from '../../util/getBuildConfig'
+
+const DownloadPage = (): ReactElement => {
+  const { t: regionSettingsTranslations } = useTranslation('regionSettings')
+  const { t: activationTranslations } = useTranslation('activation')
+
+  const config = useContext(ProjectConfigContext)
+  const { ios, android } = getBuildConfig(window.location.hostname)
+
+  return (
+    <Box display='flex' flexDirection='column' alignItems='center' mx='auto' mt={10}>
+      <Box
+        component='img'
+        sx={{
+          height: 100,
+          aspectRatio: 1,
+          marginBottom: '1rem',
```

Comment:
🔧 we try to use spacing from mui which is defined here
https://mui.com/material-ui/customization/spacing/
One step is `8px` currently as default.
We may think about adjusting that using rem. 
```suggestion
          marginBottom: 2,
```